soc/arm/Kconfig

```diff
@@ -13,3 +13,10 @@ config CPU_HAS_NXP_MPU
 	help
 	  This option is enabled when the CPU has a Memory Protection Unit (MPU)
 	  in NXP flavour.
+
+config CPU_HAS_ARM_SAU
+	bool
+	# Omit prompt to signify "hidden" option
+	select CPU_HAS_TEE
+	help
+	  MCU implements the ARM Security Attribution Unit (SAU).
```

Comment:
Platform instead of MCU?

Reply:
Here, it is really the SOC we refer to, so MCU would be fine, I suppose.